src/pipelines/epidemiology/us_nyt_covid.py

```diff
@@ -48,49 +48,95 @@ def parse_dataframes(
         data["key"] = "US_" + data["subregion1_code"]
 
         # Now that we have the key, we don't need any other non-value columns
-        data = data[["date", "key", "confirmed", "deceased"]]
-        data = grouped_diff(data, ["key", "date"])
+        data = data[["date", "key", "total_confirmed", "total_deceased"]]
         return data
 
 
+def _distribute_cases(
```

Comment:
Is this intended to be used widely, or just for a few regions where we're otherwise unable to get good data?  From what I've seen, confirmed cases as fraction of population varies widely between subregions (such as counties of California).  If we don't have good confirmed count data by subregion, then I don't think it makes sense to build subregion-based models that use count data -- they might as well just build models off the larger region.  I really don't want to oversell our data.

I can think of a couple of exceptions: if we have good subregion count data in most places and just need to fill in a few subregions, or if the models want to use a lot of other variables about the subregions and confirmed count is just one, relatively unimportant variable.

Also, if such proportional estimations are desired/requested by modelers, then as we move to cloud and space becomes less of an issue, we can think about providing a feature where a user can request not only counts but also other statistics to be estimated through a similar proportional method.